src/explore-education-statistics-common/src/modules/charts/components/LineChartLabel.tsx

```diff
@@ -0,0 +1,79 @@
+import { LineChartDataLabelPosition } from '@common/modules/charts/types/chart';
+import { LegendInlinePosition } from '@common/modules/charts/types/legend';
+import formatPretty from '@common/utils/number/formatPretty';
+import { LabelProps } from 'recharts';
+import React from 'react';
+
+interface Props extends LabelProps {
```

Comment:
Wouldn't extend `LabelProps`. It contains a whole bunch of things that we don't pass onto the final rendered element so it'd be misleading to include these.

We should just define the props `x`, `y` and `value`